crates/red_knot_python_semantic/resources/mdtest/narrow/isinstance.md

```diff
@@ -76,6 +83,15 @@ if isinstance(x, A):
     reveal_type(x)  # revealed: A
     if isinstance(x, B):
         reveal_type(x)  # revealed: A & B
+    else:
+        reveal_type(x)  # revealed: A & ~B
+
+if isinstance(x, (A, B)):
+    reveal_type(x)  # revealed: A | B
+elif isinstance(x, (A, C)):
+    reveal_type(x)  # revealed: C & ~A & ~B
+else:
+    reveal_type(x)  # revealed: object & ~A & ~B & ~C
```

Comment:
Not sure if always desirable, but we could maybe add a `TODO` here to always remove `object` from the positive side of an intersection.